data/wanderers middle.txt

```diff
@@ -1766,7 +1766,51 @@ mission "Wanderers: Sestor Search: Varu K'prai"
 			`	The Wanderers don't know what happened next, but the drones are no longer here, so they must have moved on to a different system.`
 				decline
 
-
+mission "Wanderers: Sestor Search: Human Spaceport"
+	landing
+	source
+		government Republic "Free Worlds" Syndicate Neutral
+		attributes north
+	to offer
+		has "Wanderers: Sestor Search: active"
+	on offer
+		conversation
+			`As you bring your ship in for landing at an empty pad, you notice the captain of an adjacent Behemoth inspecting his ship, which is raked with projectile punctures.`
+			choice
+				`	(Approach the captain and try to find out what happened)`
+					goto approach
+				`	(Ignore him.)
+					decline
+			label approach
+			`	You get a better view of the Behemoth——the "Heart of Gold", as indicated on the ship's stern——as you walk up to it. The rear of the ship is badly hammered, and black soot-like stains streak forward across the hull. You can't help but notice that at several projectile entry along the hull, the metal is bent smoothly inwards, almost as if the penetrator had seared or melted its way through the ship's armor. The captain of the Behemoth looks up from his inspection. Seeing you, he remarks, "We were lucky to get out of that one alive!"
+			choice
+				`	"How did your ship end up in such a state?"`
+				`	"Did those damn pirates maul this freighter to pieces?"`
+			`	"I'm not quite sure," the captain replies. "We were shot by a fleet of hostile ships that our computer couldn't identify. They didn't look at all like a run-of-the-mill pirate ship, though, and they were kitted out with tech I've never seen before."`
+			choice
+				`	"How many ships were there? What did they look like?"`
+				`	"Where did this fight take place?"`
+				`	"You should be glad you're still in one piece."
+					goto glad
+			`	"Well," the pilot continues, "we had hired a beefy Leviathan escort to Farpoint in the Alnitak system. When we were going in for landing, two of these hostile ships jumped in from the northwest, the direction of all the anarchist colonies around here. The ships weren't big, but they were tough——our four heavy laser turrets weren't enough to produce even a dent in their shields. They immediately shredded the Leviathan warship, and nearly disabled our ship, too. It was lucky I decided that jumping would be faster than landing. We warped out of the system while in atmosphere, saving the ship, but now I'm in debt to all my crew because I couldn't meet my rush delivery deadline."`
```

Comment:
"They immediately shredded the Leviathan warship"

How about just "They shredded the Leviathan"?